debezium-core/src/main/java/io/debezium/config/CommonConnectorConfig.java

```diff
@@ -693,7 +700,8 @@ public static FieldNameAdjustmentMode parse(String value) {
                     SIGNAL_ENABLED_CHANNELS,
                     TOPIC_NAMING_STRATEGY,
                     NOTIFICATION_ENABLED_CHANNELS,
-                    SinkNotificationChannel.NOTIFICATION_TOPIC)
+                    SinkNotificationChannel.NOTIFICATION_TOPIC,
+                    SOURCE_INFO_STRUCT_MAKER)
```

Comment:
@adasari This should be included in per-connector config class as the definition changes. Othrwise the tools will not see the default value in config definition.

Reply:
Left a comment - https://github.com/debezium/debezium/pull/4258/files#r1213185645. thanks.